src/site/filters/liquid.js

```diff
@@ -593,4 +589,47 @@ module.exports = function registerFilters() {
 
   liquid.filters.sortEntityMetatags = item =>
     item ? item.sort((a, b) => a.key.localeCompare(b.key)) : undefined;
+
+  liquid.filters.createEmbedYouTubeVideoURL = url => {
+    if (!url) {
+      return url;
+    }
```

Comment:
If there isn't a URL, the parent component will hopefully never call this filter. I'm not opposed to even throwing an error in this case

Reply:
I'm actually not going to throw an error, I'll just use the falsey value to conditional render the video element since they may not want to add one